fastlane/Fastfile

```diff
@@ -137,14 +142,17 @@ def attach_changelog_to_main(options)
   current_changelog_data = current_changelog.read
   master_changelog_data = master_changelog.read
 
-  current_changelog.close  
+  current_changelog.close
   master_changelog.close
 
   File.open("../CHANGELOG.md", 'w') { |master_changelog_write_mode|
     version_header = "## #{version_number}"
     whole_file_data = "#{version_header}\n\n#{current_changelog_data}\n#{master_changelog_data}"
     puts "going to save. Contents - #{whole_file_data}"
-    
     master_changelog_write_mode.write(whole_file_data)
   }
 end
+
+def is_snapshot_version?(version_name)
+    version.end_with?("-SNAPSHOT")
```

Comment:
last ruby nitpick: use 2 spaces for indents (my bad, I used 4 in the suggestions out of habit)